.dockerignore

```diff
@@ -0,0 +1,4 @@
+vendor/
+tests/
+.travis.yml
+phpunit.xml.dist
```

Comment:
you can also avoid including the readme, the box.json file and other config files. They are useless in the image. and ``/build`` should be ignored as well, in case you built a phar locally